### DISCUSSION TITLE
Chrome extension for creating Wikipedia selections

### DISCUSSION THREAD

**Discussion by audiodude** [kiwix/overview#102] [Ideas]
Given that books on Wikipedia are deader than dead (https://en.wikipedia.org/wiki/Help:Books), would it be useful to create a Chrome extension where people could "bookmark" pages as they browse Wikipedia, and later create a selection (and via Wikipedia On Demand, a ZIM)?

**Comment by kelson42** [comment]
The need is clear, but wonder if a gadget https://en.wikipedia.org/wiki/Wikipedia:Gadget would not be a better approach.
